### DISCUSSION TITLE
$generateHtmlFromNodes: Empty editor returns <p><br><p>?!?

### DISCUSSION THREAD

**Discussion by nerdess** [facebook/lexical#4459] [Q&A]
I tried this in the playground https://playground.lexical.dev/ and also locally on my Lexical instance, same odd result.

When I remove all nodes and click on "exportDOM" in the tree view I get this:

<img width="404" alt="Bildschirmfoto 2023-05-05 um 12 56 13" src="https://user-images.githubusercontent.com/711925/236440933-2117b4f1-0280-4e1d-87a0-e58923f0a439.png">

I think this is really confusing. An empty editor should return `"" `and not `"<p><br></p>"` when running `$generateHtmlFromNodes`.

Or am I missing sth?

**Comment by reactuz** [comment]
@nerdess Feels strange indeed, but that's how it's working. I've added a regex expression to replace it with empty string if that's all it contains when persisting editor to html.
